src/isc-dhcp/patch/0013-buster-build.patch

```diff
@@ -0,0 +1,99 @@
+diff --git a/includes/omapip/isclib.h b/includes/omapip/isclib.h
```

Comment:
Why is a patch file still necessary for building on Buster? Version 4.4.1 is officially available for Buster; I don't believe we should need to patch just to get the build to succeed.

Reply:
Deleted.